### ISSUE TITLE
stats_over_http: long opts do not work

### ISSUE THREAD

**Issue by poblahblahblah** [apache/trafficserver#1982]
Hello,

It looks like the long options, `--integer-counters` and `--wrap-counters`, don't actually work, although the short options, `-i` and `-w`, work as expected.

I would be more than happy to update the documentation to be accurate, or I can dig into the plugin and see if I can get long options working.

Short option plugin.config:
```
[vagrant@default-centos-72 ~]$ cat /opt/trafficserver/etc/trafficserver/plugin.config
stats_over_http.so foobar -i
```

Short option curl output: https://gist.github.com/poblahblahblah/128db91e9d501dc4e82a27c33e1b6785

Long option plugin.config:
```
[vagrant@default-centos-72 ~]$ cat /opt/trafficserver/etc/trafficserver/plugin.config
stats_over_http.so _foobar --integer-counters
```

Long option curl output: https://gist.github.com/poblahblahblah/bfd1b471f86c10759852b1b6547cb434

I also noticed that specifying a long opt *before* the path modified the modifier fails to work:

```
[vagrant@default-centos-72 ~]$ cat /opt/trafficserver/etc/trafficserver/plugin.config
stats_over_http.so --integer-counters _foobar
```

output: https://gist.github.com/poblahblahblah/437a54d4f7027bc5842217968a23dbc8

It looks like the code will accept `-integer-counters`, but not `--integer-counters`.

Tested on ATS 6.2.0 on CentOS 7.x

**Comment by poblahblahblah** [comment]
Let me know if you just want me to submit a PR for the docs, I would be more than happy to.

**Comment by bryancall** [comment]
If this is still an issue please reopen.  It looks like this has been fixed in later releases 7.0.0 and above.
